lib/voteStorage.js

```diff
@@ -113,6 +125,24 @@ api.add = (ledgerNodeId, electionTopic, vote, options, callback) => {
 };
 
 /**
+ * Determine if a vote with a given hash exists.
+ *
+ * @param ledgerNodeId the ID of the node that is tracking this vote
+ * @param voteHash the hash of the vote
+ * @param callback(err, result) called once the operation completes.
+ */
+api.exists = (ledgerNodeId, voteHash, callback) => {
+  const query = {
+    ledgerNodeId: database.hash(ledgerNodeId),
+    'meta.deleted': {$exists: false},
+    'meta.voteHash': voteHash
+  };
+  const collection = database.collections.continuity2017_vote;
+  // FIXME: is there a better/faster way?
+  collection.findOne(query, {_id: 1}, (err, result) => callback(err, !!result));
```

Comment:
If it can use an index, it should be fast. But aggregate is also an option (which would also use an index). I would think either of these would be better than `findOne` x 100, etc.